dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassPathEntry.java

```diff
@@ -76,9 +80,17 @@ public int hashCode() {
   @Override
   public String toString() {
     if (artifact != null) {
-      return "Artifact(" + artifact + ")";
+      // Group ID, artifact ID and version. No extension such as "jar" or "tar.gz", because Linkage
+      // Checker uses only JAR artifacts.
+      return Artifacts.toCoordinates(artifact);
     } else {
-      return "JAR(" + jar + ")";
+      return jar.toString();
```

Comment:
Because it's clear that `b-1.0.0.jar` is a JAR file and `foo:b:1.0.0` is a Maven artifact, I removed "Artifact" and "JAR" from toString output.